src/payment/data/actions.js

```diff
@@ -20,7 +24,7 @@ export const fetchActiveOrder = createRoutine('FETCH_ACTIVE_ORDER');
 export const addCoupon = createRoutine('ADD_COUPON');
 export const removeCoupon = createRoutine('REMOVE_COUPON');
 export const updateQuantity = createRoutine('UPDATE_QUANTITY');
-export const pollPaymentState = createRoutine('UPDATE_PAYMENT_STATE');
+export const pollPaymentState = createCustomRoutine('UPDATE_PAYMENT_STATE', ['RECEIVED']);
```

Comment:
I figured out how to make.a custom Routine Action and have replaced the manual event with the routine.